src/algebra/ring/basic.lean

```diff
@@ -970,12 +970,12 @@ protected theorem map_sub {α β} [non_assoc_ring α] [non_assoc_ring β] (f : 
 /-- A ring homomorphism is injective iff its kernel is trivial. -/
 theorem injective_iff {α β} [non_assoc_ring α] [non_assoc_semiring β] (f : α →+* β) :
```

Comment:
Could consider just removing these two lemmas now

Reply:
I suspect you'll find a matching pair for linear_map

Reply:
done, although I didn't find one for linear map. I did also remove the one for alg_hom. We'll see about CI. I had to do it all blind for obvious reasons.